src/main/java/server/response/Response.java

```diff
@@ -79,6 +79,12 @@ public Builder withLocationHeader(Request request, String path) {
             return this;
         }
 
+        public Builder withContentTypeHeader(File file) {
+            this.headers.put(HeaderFields.CONTENT_TYPE, getContentType(file));
```

Comment:
I think it's probably best if getting the content type according to the given file is done in another place. withContentTypeHeader sounds like it just takes a contentType as a param. Also instead of creating one of these methods for each header, why not just have one function that takes in the header name & value?